.github/ISSUE_TEMPLATE/crash.yml

```diff
@@ -8,6 +8,22 @@ body:
           This form is for hard crashes of the Python interpreter, segmentation faults, failed C-level assertions, and similar. Unexpected exceptions raised from Python functions in the standard library count as bugs rather than crashes.
 
           The CPython interpreter is written in a different programming language, C. A "CPython crash" is when Python itself fails, leading to a traceback in the C stack.
+  - type: textarea
+    attributes:
+      label: What happened?
+      description: >
+        Include a [minimal, reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) if possible.
+        Copy and paste code where possible rather than using screenshots,
+        and put any code blocks inside triple backticks.
+        If there were any error messages or core dumps,
+        make sure to copy them here as well.
```

Comment:
Maybe this is too obvious to need stating

Reply:
Nothing is ever too obvious...

I'd lean towards keeping, I don't think it hurts to be explicit here.

Reply:
> Nothing is ever too obvious...

Very true! However, I think it's really important to be as concise as possible in the issue forms -- if it looks like it's going to be a lot to read, I think people tend to just not read any of it

Reply:
I've removed it for now; I don't think people not pasting their core dumps counts as a major triaging issue right now, so I think this sentence is something of a solution in search of a problem. We can always add something like this sentence back if we decide in the future that this is a recurring problem :)